sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs

```diff
@@ -38,7 +38,7 @@ public ConfidentialLedgerCertificateClient(Uri certificateEndpoint, Confidential
             options ??= new ConfidentialLedgerCertificateClientOptions();
             ClientDiagnostics = new ClientDiagnostics(options);
             _pipeline = HttpPipelineBuilder.Build(options, Array.Empty<HttpPipelinePolicy>(), Array.Empty<HttpPipelinePolicy>(), new ResponseClassifier());
-            _certificateEndpoint = certificateEndpoint;
+            _endpoint = certificateEndpoint;
```

Comment:
nit: wouldn't be clearer to keep both  `_certificateEndpoint `and `_ledgerEndpoint `?

Reply:
This is due to API spec change I believe.  The other part of this class is generated from the spec. 

The stable version used `ledgerEndpoint`: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

The preview versions, I believed we had to change the name from `ledgerEndpoint` to `endpoint`: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-01-26-preview/confidentialledger.json